infrastructure/controllers/cert-manager.yaml

```
@@ -32,4 +32,6 @@ spec:
         namespace: cert-manager
       interval: 12h
   values:
-    installCRDs: true
+    crds:
+      enabled: true
+      keep: true
```

```suggestion
      keep: false
```

Make it more aligned with GitOps :)